_inc/client/components/jetpack-logo/index.jsx

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Do you think you could add a readme for that new component? That may help others who may want to use it in the future. You can look at other components to get some readme examples.

Reply:
Sounds good. Will take care of that 👍

Reply:
A simple readme file has been added in https://github.com/Automattic/jetpack/pull/13348/commits/8559fee42e85cb4b3f247e68486e62e2ca17a3a0.